lib/trino-plugin-toolkit/pom.xml

```diff
@@ -123,6 +123,12 @@
             <artifactId>jmxutils</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
I'm not sure what are the rules about scope:provided here, but I assumed that since `opentelemetry-context` is in that scope, `opentelemetry-api` should be as well. But that may be completely wrong, so please correct me if needed :)

Reply:
```
    private static final ImmutableList<String> SPI_PACKAGES = ImmutableList.<String>builder()
            .add("io.trino.spi.")
            .add("com.fasterxml.jackson.annotation.")
            .add("io.airlift.slice.")
            .add("org.openjdk.jol.")
            .add("io.opentelemetry.api.")
            .add("io.opentelemetry.context.")
            .build();
```

it is provided scope

Reply:
Cool, thanks!